tsconfig.json

```
@@ -0,0 +1,25 @@
+{
+  "compilerOptions": {
+    "module": "CommonJS",
+    "lib": [
+      "esnext"
+    ],
+    "target": "ES6",
```

Change target to be at least ES2017, or higher. With ES6, it's polyfilling Promises which isn't necessary.
```suggestion
    "target": "ES2017",
```

---

Seems to compile/test with ESNext, does that sound good?

---

Seems to compile/test with ESNext. Does that sound good?